android/gradle.properties

```diff
@@ -2,3 +2,5 @@ ReactNativeNetInfo_compileSdkVersion=28
 ReactNativeNetInfo_buildToolsVersion=28.0.3
 ReactNativeNetInfo_targetSdkVersion=27
 ReactNativeNetInfo_minSdkVersion=16
+android.useAndroidX=true
+android.enableJetifier=true
```

Comment:
I *think* that these lines are only needed for apps which want to use AndroidX, not for libraries. Can you remove them please.